Add max scan query management prop (#16792)
trinodb/trino: docs/src/main/sphinx/admin/properties-query-management.rst

time spend in a queue waiting, so essentially this is the time allowed for a
query to exist since creation.

``query.max-scan-physical-bytes``
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

* **Type:** :ref:`prop-type-data-size`

> * **Default value:** ``empty``

Reviewer: I think we just leave this out .. basically there is no default value .. so there is no defined max value ..

Reviewer: ``empty`` is not a valid value by itself

* **Session property:** ``query_max_scan_physical_bytes``

The maximum number of bytes that can be scanned by a query during its execution.

> When this limit is reached, the query will be terminated to prevent excessive

Reviewer: ```suggestion
When this limit is reached, query processing is terminated to prevent excessive
```
